src/libraries/System.DirectoryServices.Protocols/src/System/DirectoryServices/Protocols/ldap/LdapDirectoryIdentifier.cs

```diff
@@ -35,7 +35,7 @@ public LdapDirectoryIdentifier(string[] servers, bool fullyQualifiedDnsHostName,
                     if (servers[i] != null)
                     {
                         string trimmedName = servers[i].Trim();
-                        string[] result = trimmedName.Split(new char[] { ' ' });
+                        string[] result = trimmedName.Split(' ');
```

Comment:
I think this is a reasonable change. It'll bind to different overloads based on the target framework and at least save on the array allocation when built for core.